docs/logger-mdc-instrumentation.md

```diff
@@ -28,11 +28,11 @@ logs can correlate traces/spans with log statements.
 
 > Note: There are also log appenders for exporting logs to OpenTelemetry, not to be confused with the MDC appenders.
 
-| Library | Auto-instrumented versions | Standalone Library Instrumentation                                                   |
-|---------|----------------------------|--------------------------------------------------------------------------------------|
-| Log4j 1 | 1.2+                       |                                                                                      |
-| Log4j 2 | 2.7+                       | [opentelemetry-log4j-context-data-2.17-autoconfigure](../instrumentation/log4j/log4j-context-data/log4j-context-data-2.17/library-autoconfigure) |                                                                                     |
-| Logback | 1.0+                       | [opentelemetry-logback-mdc-1.0](../instrumentation/logback/logback-mdc-1.0/library)  |
+| Library | Auto-instrumented versions | Standalone Library Instrumentation |
+|-|-|-|
+| Log4j 1 | 1.2+ | [MDC](../instrumentation/log4j/log4j-mdc-1.2/) |
+| Log4j 2 | 2.7+ | [ContextData](../instrumentation/log4j/log4j-context-data/log4j-context-data-2.17/library-autoconfigure/README.md) |
+| Logback | 1.0+ | [MDC](../instrumentation/logback/logback-mdc-1.0/javaagent/README.md) |
```

Comment:
The links in the 3rd column are describing the standalone library instrumentations; I don't think that the changes you've made are correct (we don't have library instrumentation for log4j1, and the logback link points to the javaagent module for some reason).
Can you revert changes in this file?